scripts/install.js

```diff
@@ -8,6 +8,7 @@ copy('../dist/config.xml', '../config.xml');
 copy('../dist/config.scss', '../config/config.scss');
 copy('../dist/index.js', '../config/index.js', true);
 copy('../dist/templates', '../config/templates');
+copy('../dist/logo.png', '../img/logo.png', true);
```

Comment:
sorry if it was not clear but if you do not want git to track this file, it need to live in `config`:

```
copy('../dist/logo.png', '../config/logo.png', true);
```

thanks